docs/developers/developing_iree/profiling_with_tracy.md

```diff
@@ -241,6 +241,24 @@ Internally, Tracy executes `su` commands to perform certain actions, so it too
 relies on the device being *rooted* without relying on the benchmark process
 being run as root.
 
+## "RESOURCE_EXHAUSTED; failed to open file" issue
+
+This is a known issue with how tracy operates.
```

Comment:
Please link to the Tracy issue you filed.

Reply:
Done!